packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -142,9 +146,8 @@
   [aria-expanded='true'] & {
     background-color: var(--usa-banner-close-button__background-color--mobile);
     display: flex;
-    inset: 0;
+    inset: 0.25em 0.25em 0.25em 0;
```

Comment:
Removing min-height and changing `inset` to accommodate the offset value allows the close button, which only appears on mobile when open, reveals the focus ring around the entire component.

Reply:
A possibly unintended side-effect of this insetting is that the unfocused button looks different now. The 'X' button loses its square shape and gets moved.

<img width="548" alt="Before" src="https://github.com/CMSgov/design-system/assets/7595652/76588d67-a8d0-412e-bb0d-d5bd01a3a331">

<img width="548" alt="After" src="https://github.com/CMSgov/design-system/assets/7595652/e12822dc-e414-4b29-8e8f-a8c4fcc8a3de">

What if we applied the outline to an absolutely positioned pseudoelement that could render over the top of that 'X' span?

Reply:
@zarahzachz, it looks to me like [the border strategy](https://github.com/CMSgov/design-system/pull/3030/commits/9883d56ef15a0c804525519ad4fea44676e5a179) has the same side-effect of making the X button rectangular and inset

Reply:
Never mind! I forgot to pull!

Reply:
<img width="124" alt="Screenshot 2024-04-15 at 9 22 31 AM" src="https://github.com/CMSgov/design-system/assets/7595652/54c8b7c3-fbbe-4bb1-91ef-3b8d4dc84768">

there is still this subtle shift of the X to the left inside the button, but this is a more elegant solution than the one I had proposed